ere-guests/stateless-validator/reth/zisk/Cargo.toml

```diff
@@ -22,7 +22,10 @@ reth-primitives-traits = { workspace = true, features = [
 reth-evm-ethereum.workspace = true
 reth-chainspec.workspace = true
 
-revm = { workspace = true, default-features = false, features = ["bn"] }
+revm = { workspace = true, default-features = false, features = [
+    "kzg-rs",
```

Comment:
@han0110 , reg [this comment](https://github.com/eth-act/zkevm-benchmark-workload/pull/178#discussion_r2335521511), we can't disable it since we have to provide at least one of `c-kzg` or `kzg-rs`, thus why all zkVMs use `kzg-rs` so can be patched with a precompile.

I think this means we're back again to what this https://github.com/eth-act/zkevm-benchmark-workload/pull/116 was referring to.

Reply:
https://github.com/eth-act/zkevm-benchmark-workload/issues/181

Reply:
> but the benchmark case stressing the KZG proof verif precompile will be a huge killer for a "bad reason".

I think ZisK's patch of `bls12_381` is only used to disable sp1's features, it still uses the software impl in `kzg-rs` crate, so if the `revm` fallback impl is not much worse than `kzg-rs`'s software impl, then perhaps we could just use the fallback to reduce the patching complexity.